mvp-spec/operator-api.md

```diff
@@ -268,52 +266,48 @@ timestamp
 
 #### Request
 
-<!-- The query string below is generated with taking the request-cmpC.json file, removing body, and encoding it as query string:
-npx encode-query-string -nd `cat request-cmpC.json | npx json -e 'this.body = undefined' -o json-0`
--->
-
 ```http
 POST /v1/id-prefs
 ```
 
 ##### Request payload
 
-<!-- Update this code block with just taking the body of body-id-and-preferences.json:
-cat request-cmpC.json body-id-and-preferences.json | npx json --merge
--->
-
+<!--partial-begin { "files": [ "paf-mvp-demo-express/src/examples/generated-examples/postIdPrefsRequest.json" ], "block": "json" } -->
```

Comment:
Can we manage to have all the partials in one place?

Reply:
Not for the moment: they are generated by the demo application, not by this project.
We could copy them to the `partials` directory, but I don't think it would add any value, don't you think?
Note that I put the git submodule in `partials`, already.

Reply:
After discussion, reduced path with symbolic link